packages/core/nuxt-theme-module/theme/components/NewsletterModal.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+  <SfModal
+    :visible="isNewsletterModalOpen"
+    class="modal"
+    @close="closeModal"
+  >
+    <template #modal-bar>
+      <SfBar
+        class="modal__title smartphone-only"
+        :close="true"
+        :title="$t('Subscribe to newsletter')"
+        @click:close="closeModal"
+      />
+    </template>
+    <transition name="sf-fade" mode="out-in">
+      <div>
+        <SfHeading
+          :level="3"
+          :title="$t('Subscribe to newsletter')"
+          class="modal__title desktop-only"
+        />
+        <form  @submit.prevent="$emit('email', emailAddress)">
+          <SfInput
+            type="email"
+            label="Email address"
+            v-model="emailAddress"
+            class="modal__input"
+          />
+          <SfButton class="modal__button" type="submit">
+            I confirm subscription
+          </SfButton>
+        </form>
+        <SfHeading
+          description="You can unsubscribe at any time"
+          :level="3"
+        />
+        <SfScrollable :maxContentHeight="isMobile ? 'auto' : '3.75rem'" :class="{ 'is-open': !isHidden }">
+          <p class="modal__content">
+            After signing up for the newsletter, you will receive special offers and messages from VSF via email.
```

Comment:
Here also the translation. For the SfLink you can use https://kazupon.github.io/vue-i18n/guide/interpolation.html#basic-usage.

Reply:
https://github.com/vuestorefront/vue-storefront/pull/5971/files#r648038706

Reply:
Thank for the tip :)